src/util/internalCallTree.js

```diff
@@ -36,6 +36,7 @@ class InternalCallTree {
             this.event.trigger('callTreeBuildFailed', [result.error])
           } else {
             console.log('ready')
+            this.reducedTrace.push(traceManager.trace.length - 1)
```

Comment:
I think it would be better to put this line into some kind of `createReducedTrace` function.